src/porepy/models/solution_strategy.py

```diff
@@ -449,7 +449,9 @@ def after_nonlinear_convergence(
             values=solution, time_step_index=0, additive=False
         )
         self.convergence_status = True
-        self.save_data_time_step()
+
+        times_to_export = self.params.get("times_to_export", None)
+        self.export_solution(times_to_export=times_to_export)  # type: ignore
```

Comment:
I think we should keep using save_data_time_step and keep the method in the visualization part of the model.